rholang/src/main/scala/coop/rchain/rholang/interpreter/package.scala

```diff
@@ -1,11 +1,14 @@
 package coop.rchain.rholang
 
 import cats.effect.Bracket
+import coop.rchain.rholang.interpreter.RhoRuntimeSyntax
 
 package object interpreter {
 
   type _error[F[_]] = Bracket[F, Throwable]
 
   def _error[F[_]](implicit ev: _error[F]): _error[F] = ev
+  object syntax extends AllSyntaxRholang
 
 }
+trait AllSyntaxRholang extends RhoRuntimeSyntax
```

Comment:
I'm concerned how developers can navigate thru different _syntax_ definitions and it make sense that each project has its own _all_ definition.
So these definitions are more for `rholang/package.scala` file.
Import is then `coop.rchain.rholang.syntax._` similar to e.g. casper `coop.rchain.casper.syntax._`.

Reply:
Ok. After some tries, I understand why I put the syntax in `coop.rchain.rholang.interpreter.syntax._` way because there are auto-generated java codes by bnfc in `coop.rchain.rholang.interpreter.syntax` package.

I think the syntax meaning for these two things are different and ambiguous so I put it in the `interpreter` package.

Do you still think I should put it in `coop.rchain.rholang.syntax._`this way?

Reply:
I see, this is unfortunate. Maybe we could rename generated namespace to _parser_ instead of _syntax_.
https://github.com/rchain/rchain/blob/116c21bf647bc634251b13bd8021487663b560fb/project/BNFC.scala#L95

Renaming does not involve to many files but better to do this on dev branch and with this PR. Please just add this task as part of cleanup issue #3001.